src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -48,17 +49,32 @@ class CosmasGoogleCloudService(private val bucketName: String,
         val COSMAS_ID = "robot:::cosmas"
     }
 
+    fun bucketName(info: ProjectInfo): String = if (info.isFreePlan) this.freeBucketName else this.paidBucketName
+
+    fun fileStorageName(fileId: String, info: ProjectInfo): String = info.ownerId + "/" + fileId
```

Comment:
We don't have to create a directory prior to placing a file into it?

Reply:
"If the destination object ends with a "/" gsutil treats it as a directory"
https://cloud.google.com/storage/docs/gsutil/addlhelp/HowSubdirectoriesWork